src/configs/config.mainnet.js

```diff
@@ -10,6 +10,16 @@ const blockExplorerUrlLiquid = isTor
     ? "http://liquidmom47f6s3m53ebfxn47p76a6tlnxib3wp6deux7wuzotdr6cyd.onion"
     : "https://liquid.network";
 
+export const torUrl =
+    "http://boltzzzbnus4m7mta3cxmflnps4fp7dueu2tgurstbvrbt6xswzcocyd.onion/";
+export const discordUrl = "https://discord.gg/QBvZGcW";
+export const twitterUrl = "https://twitter.com/boltzhq";
+export const githubUrl = "https://github.com/BoltzExchange";
+export const docsUrl = "https://docs.boltz.exchange";
+export const blogUrl = "https://blog.boltz.exchange";
+export const nostrUrl =
+    "https://snort.social/p/npub1psm37hke2pmxzdzraqe3cjmqs28dv77da74pdx8mtn5a0vegtlas9q8970";
```

Comment:
How about we don't copy paste those 3 times but import them similarly to how we did it wit the beta config file?